fix 增加9月常见问题， (#726)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/start/overview.mdx

- **[TapTap Login](/sdk/taptap-login/features/)**: Allows players to quickly log into the game by authorizing with a TapTap account.

> - **[TapTap Connect](/sdk/taptap-connect/features/)**: Provides a tool to connect TapTap, game and player. Player who have successfully signed in to their TapTap accounts can browse game news, rate games, share game download links and more in the hover window to help games improve retention and attract new user.

Reviewer: ```suggestion
- **[TapTap Connect](/sdk/taptap-connect/features/)**: A tool that connects TapTap, the game, and the player. Players who have successfully signed in to their TapTap accounts can browse game news, rate games, share game download links and more in TapTap Connect, which helps games improve retention and attract new users.

```

- **[TapTap Login](/sdk/taptap-login/features/)**: Allows players to quickly log into the game by authorizing with a TapTap account.

> - **[TapTap Connect](/sdk/taptap-connect/features/)**: A tool that connects TapTap, the game, and the player. Players who have successfully signed in to their TapTap accounts can browse game news, rate games, share game download links and more in TapTap Connect, which helps games improve retention and attract new users.

Author: ```suggestion
- **[TapTap Connect](/sdk/taptap-connect/features/)**: A tool that connects TapTap, the game, and the player. 
```

Author: 这个英文我也同步改下

- **[Achievements](/sdk/achievement/features/)**: Allows you to set up “Achievements” and “Platinum Achievements” to increase player participation and encourage players to explore different ways to play the game.

- **[Billboard](/sdk/billboard/features/)**: Billboard allows developers to publish and edit billboard contents on the Developer Center, which players can see when they open the game.

</Conditional>

- **[Leaderboards](/sdk/leaderboard/features/)**: Based on TDS Authentication, you can set up leaderboards in your game to promote competition among players and thereby increase player activity.

- **[Cloud Save](/sdk/gamesaves/features/)**: Saves the player’s game progress to the TDS server, where the game can retrieve saved game data and allow the player to continue playing from any save point on any device.

- **[Gift Packages](/sdk/tds-gift/features/)**: Gift Packages lets you create gift codes for your game which players can redeem in your game later.

> - **[TapLink](/sdk/taplink/features/)**: Help player jump from TapTap to the game to collect the gift packs.

Reviewer: ```suggestion
- **[TapLink](/sdk/taplink/features/)**: Helps players jump from TapTap to the game to collect the gift packs.

```

- **[Push Notification](/sdk/push/features/)**: A unified service that integrates push notifications for both Android and iOS.

- **[Multiplayer](/sdk/multiplayer/features/)**: Rely on cloud services for easy in-game player matching, online matchmaking message synchronisation and more.

> - **[Customer Support](/sdk/tap-support/features/)**: Access to Tap Game Customer Service helps the game operation team to solve problem encountered by player faster and better.

Reviewer: ```suggestion
- **[Customer Support](/sdk/tap-support/features/)**: Helps the game's operation team better solve the problems encountered by players.

```
